libappstream-glib/as-self-test.c

```diff
@@ -1722,6 +1725,15 @@ as_test_content_rating_func (void)
 			 AS_CONTENT_RATING_VALUE_MILD);
 	g_assert_cmpint (as_content_rating_get_value (content_rating, "violence-bloodshed"), ==,
 			 AS_CONTENT_RATING_VALUE_UNKNOWN);
+
+	rating_ids = as_content_rating_get_rating_ids (content_rating);
+	g_assert_nonnull (rating_ids);
+
+	for (i = 0; rating_ids[i] != NULL && expected_rating_ids[i] != NULL; i++)
```

Comment:
`for (gsize i = 0; ` pls

Reply:
Not possible, since `i` is needed outside the loop to check that both lists have terminated (with `NULL`) at the same time.

Reply:
/me gets more coffee -- thanks.